go/src/koding/kites/kloud/utils/object/object.go

```diff
@@ -322,6 +329,15 @@ func isMapObject(v interface{}) bool {
 	return typ.Key().Kind() == reflect.String
 }
 
+func isNil(v interface{}) bool {
+	switch v := reflect.ValueOf(v); v.Type().Kind() {
+	case reflect.Chan, reflect.Func, reflect.Map, reflect.Ptr, reflect.Slice:
```

Comment:
TIL:

this works https://play.golang.org/p/q6NZKo5KUo

but 

this doesnt https://play.golang.org/p/zdOCtq4x9g